validator/lib/validateI3dm.js

```diff
@@ -30,31 +39,60 @@ function validateI3dm(content) {
     if (magic !== 'i3dm') {
         return {
             result : false,
-            message: 'Header has an invalid magic field. Expected version = \'i3dm\'. Found magic = ' + magic
+            message: 'i3dm tile has an invalid magic field. Expected version = \'i3dm\'. Found magic = ' + magic
         };
     }
 
     if (version !== 1) {
         return {
             result : false,
-            message: 'Header has an invalid version field. Expected version = 1. Found version = ' + version
+            message: 'i3dm tile has an invalid version field. Expected version = 1. Found version = ' + version
         };
     }
 
     if (byteLength !== content.length) {
         return {
             result : false,
-            message: 'Header has an invalid byteLength field. Expected byteLength = ' + content.length + '. Found byteLength = ' + byteLength
+            message: 'i3dm tile has an invalid byteLength field. Expected byteLength = ' + content.length + '. Found byteLength = ' + byteLength
         };
     }
 
     if (gltfFormat !== 0 && gltfFormat !== 1) {
         return {
             result : false,
-            message: 'Header has an invalid gltfFormat field. Expected gltfFormat = 0 or 1. Found gltfFormat = ' + gltfFormat
+            message: 'i3dm tile has an invalid gltfFormat field. Expected gltfFormat = 0 or 1. Found gltfFormat = ' + gltfFormat
         };
     }
 
+    var offset = 32;
+    var featureTableJSONByteLength = content.readUInt32LE(12);
+    var featureTableBinaryByteLength = content.readUInt32LE(16);
+    offset += featureTableJSONByteLength + featureTableBinaryByteLength;
```

Comment:
Style: move this a few lines below so it doesn't cut off the header value reads.